portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,92 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Caveat&family=Montserrat&family=Nunito+Sans&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+  <body class="home">
```

Comment:
I think this file could benefit from some whitespace to break up the sections. its a bit hard to read currently.

Reply:
sounds good